draft/0000-name-main-command-django.rst

```diff
@@ -0,0 +1,184 @@
+========================================
+DEP XXXX: Name the main command `django`
+========================================
+
+:DEP: XXXX
+:Author: Ryan Hiebert
+:Implementation Team: Ryan Hiebert
+:Shepherd: Tom Carrick
+:Status: Draft
+:Type: Feature
+:Created: 2025-01-07
+
+.. contents:: Table of Contents
+   :depth: 3
+   :local:
+
+
+Abstract
+========
+
+Motivated by a desire to remove confusing papercuts in Django
+and to follow common convention in the Python ecosystem,
+this DEP proposes to add a new ``django`` command equivalent to
+the existing ``django-admin`` command,
+and to update the documentation to prefer this new spelling.
+
+Specification
+=============
+
+The ``django`` command will be added as the preferred spelling
+for the existing ``django-admin`` command.
+
+.. code-block:: diff
+
+   --- a/pyproject.toml
+   +++ b/pyproject.toml
+   @@ -45,2 +45,3 @@
+    [project.scripts]
+   +django = "django.core.management:execute_from_command_line"
+    django-admin = "django.core.management:execute_from_command_line"
+
+Official documentation will be updated
+to reference this new ``django`` command
+everywhere that ``django-admin`` is currently referenced.
+The bulk of documentation should be be updated at once
+to ensure that the documentation gives a consistent message.
```

Comment:
It's _possible_ that it would be good to have some kind of plan for doing this. Even if it's quite simple (find `django-admin` in the docs directory and update accordingly.